app/hyperterm.js

```diff
@@ -270,21 +273,12 @@ export default class HyperTerm extends Component {
   }
 
   changeFontSize (value) {
-    const uid = this.state.sessions[this.state.active];
-    const term = this.refs[`term-${uid}`];
-    if (term) {
-      const size = term.term.prefs_.get('font-size');
-      term.term.prefs_.set('font-size', size + value);
-    }
+    this.setState({fontSize: this.state.fontSize + value});
   }
 
   resetFontSize () {
-    const uid = this.state.sessions[this.state.active];
-    const term = this.refs[`term-${uid}`];
-    if (term) {
-      // TODO: once we have preferences, we need to read from it
-      term.term.prefs_.set('font-size', 12);
-    }
+    // TODO: once we have preferences, we need to read from it
+    this.setState({fontSize: 12});
```

Comment:
Spacing after `{` + `}`. I should clarify in the README that we follow `standard` by @feross